src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/refreshanalyzer/RefreshSearchAnalyzerActionIT.kt

```diff
@@ -21,13 +21,15 @@ import org.elasticsearch.client.Request
 import org.elasticsearch.common.io.Streams
 import org.elasticsearch.common.settings.Settings
 import org.elasticsearch.common.xcontent.XContentType
+import org.junit.Assume
 import java.io.InputStreamReader
 import java.nio.charset.Charset
 import java.nio.charset.StandardCharsets
 import java.nio.file.Files
 
 class RefreshSearchAnalyzerActionIT : IndexManagementRestTestCase() {
     fun `test index time analyzer`() {
+        Assume.assumeTrue(isLocalTest)
```

Comment:
A cleaner approach - This check can be added to a common function, which can then run before every test in this file with `@Before`

```
  @Before
  public void checkIfLocalCluster() {
    Assume.assumeTrue(isLocalTest)
  }
```